### ISSUE TITLE
Can't build iOS after migration to v2 tealium-react-native

### ISSUE THREAD

**Issue by testrobpoc** [Tealium/tealium-react-native#115]
I have migrated from v1 to v2 `tealium-react-native`. And seems i got an error when building the app from Xcode.
I've tried v2.2.0, 2.1.3, 2.1.0 but I still getting the same error.

```
Undefined symbols for architecture x86_64:
  "_swift_getOpaqueTypeMetadata", referenced from:
      (extension in TealiumSwift):SwiftUI.View.trackingAppOpenUrl() -> some in libtealium-swift.a(TealiumAppTrackable.o)
      TealiumSwift.TealiumAppTrackable.body.getter : some in libtealium-swift.a(TealiumAppTrackable.o)
      l_get_witness_table 7SwiftUI4ViewRzlqd__AaBHD2_AaBPAAE22onContinueUserActivity_7performQrSS_ySo06NSUserG0CctFQOyAcAE0D7OpenURLAEQry10Foundation0K0Vc_tFQOyx_Qo__Qo_HO in libtealium-swift.a(TealiumAppTrackable.o)
  "_swift_getOpaqueTypeConformance", referenced from:
      (extension in TealiumSwift):SwiftUI.View.trackingAppOpenUrl() -> some in libtealium-swift.a(TealiumAppTrackable.o)
      TealiumSwift.TealiumAppTrackable.body.getter : some in libtealium-swift.a(TealiumAppTrackable.o)
      associated type witness table accessor for Body : SwiftUI.View in TealiumSwift.TealiumAppTrackable<A> : SwiftUI.View in TealiumSwift in libtealium-swift.a(TealiumAppTrackable.o)
      l_get_witness_table 7SwiftUI4ViewRzlqd__AaBHD2_AaBPAAE22onContinueUserActivity_7performQrSS_ySo06NSUserG0CctFQOyAcAE0D7OpenURLAEQry10Foundation0K0Vc_tFQOyx_Qo__Qo_HO in libtealium-swift.a(TealiumAppTrackable.o)
```

### Environment
```
react: 17.0.1
react-native:0.64.0
tealium-react-native: 2.2.0

macOS: 11.6.5 BigSur
Xcode: 13.2.1
```

### Pod file
```
require_relative '../node_modules/react-native/scripts/react_native_pods'
require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'

platform :ios, '11.0'

project 'MockApp',
  'Debug' => :debug,
  'Release' => :release,
  'Drafts.Debug' => :debug,
  'Drafts.Release' => :release,
  'Production.Debug' => :debug,
  'Production.Release' => :release

target 'MockApp' do
  config = use_native_modules!

  use_react_native!(
    :path => config[:reactNativePath],
    # to enable hermes on iOS, change `false` to `true` and then install pods
    :hermes_enabled => false
  )

  # Tealium
  pod "tealium-react-native-swift", :path => '../node_modules/tealium-react-native/tealium-react-native-swift.podspec'
```

I followed all instructions detailed herehttps://docs.tealium.com/platforms/react-native/install/ 
I had no issues with v1 of `telaium-react-native`. It was running under the same environment setup (same Xcode, same react native versions and etc)

Thank You!

Thank you

**Comment by testrobpoc** [comment]
Hey @jameskeith 
Thanks for your response.
I have tried to update my RN version to 0.66.0 and after that I was still getting the same error in Xcode.
After that I carefully double checked the  changes needed in [React Native Upgrade Helper](https://react-native-community.github.io/upgrade-helper) and seems I fount what I've missed.

Seems that this line was missing in my `ios/MyApp.xcodeproj/project.pbxproj`
![Screenshot 2022-03-22 at 22 13 47](https://user-images.githubusercontent.com/43346644/159568649-a53f4aa3-d72c-4539-b146-6c41fae286bb.png)

Once I added these lines, and re-build the app from Xcode. It all succeed.
